doc/introduction/measurements.rst

```diff
@@ -154,6 +154,71 @@ so corresponds to a :math:`99.75\%` probability of measuring
 state :math:`|00\rangle`, and a :math:`0.25\%` probability of
 measuring state :math:`|01\rangle`.
 
+Mid-circuit measurements and conditional operations
+---------------------------------------------------
+
+PennyLane allows specifying measurements in the middle of the circuit.
+Operations can then be conditioned on the measurement outcome of such
+mid-circuit measurements:
+
+.. code-block:: python
+
+    def my_quantum_function(x, y):
+        qml.RY(x, wires=1)
+        qml.CNOT(wires=[0, 1])
+        m_0 = qml.measure(1)
+
+        qml.cond(m_0, qml.RY)(y, wires=0)
+        return qml.probs(wires=[0])
+
+A quantum function with mid-circuit measurements (defined using
+:func:`~.pennylane.measure`) and conditional operations (defined using
+:func:`~.pennylane.cond`) can be executed by applying the deferred
+measurement principle. In the example above, we apply the ``qml.RY`` rotation
```

Comment:
Could link to https://en.wikipedia.org/wiki/Deferred_Measurement_Principle

Reply:
Thanks, added!